src/components/ButtonWithHeading/index.js

```diff
@@ -9,9 +9,9 @@ export const ButtonWithHeading = ({ heading, buttonText, onClick = () => {}}) =>
       <Container>
         <Row>
           <Col> 
-            <h1>{heading}</h1>
+            <h1>{!heading || (heading)}</h1>
```

Comment:
Don't you think a cleaner way to null check would be { heading && (heading) }

Reply:
OK, will do the changes, thanks.